content/realtime/versions/v1.0/authentication.textile

```diff
@@ -79,7 +79,7 @@ h2(#token-authentication). Token Authentication
+*Token Authentication* uses an Ably-compatible token to authenticate with Ably. This can be an "Ably Token":/core-features/authentication#tokens obtained via the REST API <span lang="default">"@requestToken@":#request-token</span><span lang="csharp">"@RequestToken@":#request-token</span><span lang="ruby">"@request_token@":#request-token</span> endpoint, an "Ably JWT":/core-features/authentication#ably-jwt signed by your API key, or an "External JWT":https://jwt.io object "with an embedded Ably-compatible token":/core-features/authentication#jwt-embed-process. Tokens are authentication credentials that are short-lived, and therefore they may more readily be distributed to clients where there is a risk of compromise. Tokens may also be issued with a particular scope - such as a limited set of "access rights or capabilities":/core-features/authentication#capabilities-explained or being limited to use by a specific "<span lang="default">@clientId@</span><span lang="csharp">@ClientId@</span> identity":#identified-clients - and therefore token-based authentication provides the flexibility to implement access and identity control policies in the application. See the "Token Authentication documentation":/core-features/authentication#token-authentication for more details. To initialize the "Realtime library":/api/realtime-sdk/constructor to a previously obtained "@Ably Token@":#token-details or "@Ably JWT@":/core-features/authentication#ably-jwt,  set the <span lang="ruby">@:token@ or @:token_details@</span><span lang="python">@token@ or @token_details@</span><span lang="csharp">@Token@ or @TokenDetails@</span><span lang="default">@token@ or @tokenDetails@</span> attribute of "@ClientOptions@":/api/rest-sdk#client-options to said token.
```

Comment:
```suggestion
*Token Authentication* uses an Ably-compatible token to authenticate with Ably. This can be an "Ably Token":/core-features/authentication#tokens obtained via the REST API <span lang="default">"@requestToken@":#request-token</span><span lang="csharp">"@RequestToken@":#request-token</span><span lang="ruby">"@request_token@":#request-token</span> endpoint, an "Ably JWT":/core-features/authentication#ably-jwt signed by your API key, or an "External JWT":https://jwt.io object "with an embedded Ably-compatible token":/core-features/authentication#jwt-embed-process. Tokens are authentication credentials that are short-lived, and therefore they may more readily be distributed to clients where there is a risk of compromise. Tokens may also be issued with a particular scope - such as a limited set of "access rights or capabilities":/core-features/authentication#capabilities-explained or being limited to use by a specific "<span lang="default">@clientId@</span><span lang="csharp">@ClientId@</span> identity":#identified-clients - and therefore token-based authentication provides the flexibility to implement access and identity control policies in the application. See the "Token Authentication documentation":/core-features/authentication#token-authentication for more details. To initialize the "Realtime library":/api/realtime-sdk to a previously obtained "@Ably Token@":#token-details or "@Ably JWT@":/core-features/authentication#ably-jwt,  set the <span lang="ruby">@:token@ or @:token_details@</span><span lang="python">@token@ or @token_details@</span><span lang="csharp">@Token@ or @TokenDetails@</span><span lang="default">@token@ or @tokenDetails@</span> attribute of "@ClientOptions@":/api/rest-sdk#client-options to said token.
```